src/site/resources/httpcomponents-core-5.1.x/documentation.links

```diff
@@ -0,0 +1,7 @@
+# links property file for Ant's symlink task in pom.xml:
+# links to components in http://hc.apache.org/components/
+LATEST   ../components/httpcomponents-core-5.1.x/LATEST
+
+# sample future release, once it has been released and
+# its documentation has been versioned from LATEST staging with something like "svn cp https://svn.apache.org/repos/asf/httpcomponents/site/components/httpcomponents-core-5.1.x/LATEST https://svn.apache.org/repos/asf/httpcomponents/site/components/httpcomponents-core-5.1.x/5.1-beta-3"
+5.1-beta-3   ../components/httpcomponents-core-5.1.x/5.1-beta-3
```

Comment:
@hboutemy I do not follow. Where would `../components/httpcomponents-core-5.1.x/5.1-beta-3` and  `./components/httpcomponents-client-5.0.x/5.0.3` come from?

Reply:
You need to copy from LATEST when you staged a release to the current version after the vote has passed. See: https://maven.apache.org/developers/website/deploy-component-reference-documentation.html#publishing-versioned-component-reference-documentation